packages/kbn-test/src/functional_tests/lib/errors.js

```diff
@@ -0,0 +1,11 @@
+const $isCliError = Symbol('isCliError');
+
+export function createCliError(message) {
+  const error = new Error(message);
+  error[$isCliError] = true;
+  return error;
+}
+
+export function isCliError(error) {
+  return error && !!error[$isCliError];
+}
```

Comment:
File completely unmodified from x-pack-kibana

Reply:
how about we keep the file, remove the `createCliError`?

Reply:
Hmm, what's the purpose if `isCliError` will always returns `false` then? Or am I missing something?

Reply:
D'oh! is that really the case? hm. Thanks for seeing that. Changing.

Reply:
At first I was thinking that you were trying to use this method for CLI error that can come from `proc_runner`, but I see it tags its error with its own `Symbol('isCliError')` that isn't the same one we have here (unless we start using `Symbol.for('isCliError')` in both packages).

Reply:
Looks like it was never used? https://github.com/elastic/x-pack-kibana/search?utf8=%E2%9C%93&q=createclierror&type=

Reply:
Looks like it was used when `proc_runner` wasn't in a separate package: https://github.com/elastic/x-pack-kibana/commit/967ded335794c6bbbd6291d247ec6e3b6e302870

Reply:
Aha, perfect. So, good to remove.